src/style.js

```diff
@@ -196,3 +198,21 @@ export function translate(x, opt_y) {
 export function scale(value) {
   return `scale(${value})`;
 }
+
+
+/**
+ * Escapes an ident (ID or a class name) to be used as a CSS selector.
+ *
+ * See https://drafts.csswg.org/cssom/#serialize-an-identifier.
+ *
+ * @param {!Window} win
+ * @param {string} ident
+ * @return {string}
+ */
+export function escapeCssSelectorIdent(win, ident) {
+  if (win.CSS && win.CSS.escape) {
+    return win.CSS.escape(ident);
+  }
+  // Polyfill.
+  return cssEscape(ident);
```

Comment:
@jridgewell So, this is actually a rare-use utility and we wanted to avoid adding it to core. That's why I removed the polyfill code from here.